lnrpc/walletrpc/walletkit_server.go

```diff
@@ -268,7 +270,19 @@ func (w *WalletKit) PublishTransaction(ctx context.Context,
 		return nil, err
 	}
 
-	err := w.cfg.Wallet.PublishTransaction(tx)
+	if len(req.Label) > wtxmgr.TxLabelLimit {
```

Comment:
We can move `getLabel` to this package to use it here and import it within the main RPC server.

Reply:
Moved to labels package, need to get around build tags to put it in here.